README.md

```diff
@@ -115,9 +115,7 @@ Your environment must meet the following requirements:
 - CUDA version >= 11.2
 - NVIDIA GPU with compute capability 8.0, 8.6, 8.7 or 9.0
 
-
-## Benchmark & Performance(W.I.P)
-
+## Benchmark
```

Comment:
Why remove "Work in Progress"? I think there are some goals to achieve in the future on the Performance.

Reply:
You're right, and I mistake in removing it. Let me add back.